src/Tizen.NUI/src/public/BaseComponents/VectorGraphics/LinearGradient.cs

```diff
@@ -49,7 +49,7 @@ internal LinearGradient(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr
         /// </summary>
         /// <param name="firstPoint">The first point used to determine the gradient bounds.</param>
         /// <param name="secondPoint">The second point used to determine the gradient bounds.</param>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
not easy for me "a surface constrained by two parallel lines crossing
        /// the given points firstPoint and secondPoint, respectively." ......